app/models/review.rb

```diff
@@ -49,7 +49,8 @@ def take_review_action
       else
         rule.review_requestor_id = nil
         rule.locked = false
-        rule.save
+        # Must skip validation here because of error_if_under_review_callback
+        rule.save(validate: false)
```

Comment:
This seems like a bad idea, there has to be a way to structure `error_if_under_review` to handle this case properly.